jetson/percep/pcl.hpp

```diff
@@ -67,13 +77,16 @@ class PCL {
         void FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices, std::vector<std::vector<int>> &interest_points);
         
         //Finds a clear path given the obstacle corners
-        double FindClearPath(std::vector<std::vector<int>> interest_points,
+        double FindClearPath(std::vector<std::vector<int>> &interest_points,
```

Comment:
Can you clarify what interest_points are? I know this is clear to you most likely but eventually you'll graduate and others will need to maintain this code.